### PR TITLE
Add 96% ANSI Community Layouts

### PR DESCRIPTION
**Pull request by noroadsleft**
## Description

[title]

## Types of Changes

- [x] Enhancement/optimization

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Review by fauxpark — COMMENTED** [review]
Looks good other than mixed spaces/NBSPs in the box drawings.

**Comment by noroadsleft** [comment]
> Looks good other than mixed spaces/NBSPs in the box drawings.

Not sure how *that* happened. :confused:

File: layouts/default/96_iso/default_96_iso/keymap.c [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+     * ├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┴───┼───┼───┼───┼───┤
+     * │ ` │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ - │ = │ Backsp│Num│ / │ * │ - │
+     * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┼───┼───┼───┼───┤
+     * │ Tab │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │     │ 7 │ 8 │ 9 │   │
+     * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐ Ent├───┼───┼───┤ + │
+     * │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │ # │    │ 4 │ 5 │ 6 │   │
+     * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴┬───┼───┼───┼───┼───┤
+     * │Sft │ \ │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │Shift │ ↑ │ 1 │ 2 │ 3 │   │
+     * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴──┬┴──┬┴──┬───┼───┼───┼───┼───┤Ent│
+     * │Ctrl│GUI │Alt │         Space          │Alt│GUI│Ctl│ ← │ ↓ │ → │ 0 │ . │   │
```
Comment (fauxpark) [inline layouts/default/96_iso/default_96_iso/keymap.c:32]: And in these ones, sorry.

Comment (noroadsleft) [reply layouts/default/96_iso/default_96_iso/keymap.c:32]: I'm not seeing any non-breaking spaces in the block you marked.

Comment (fauxpark) [reply layouts/default/96_iso/default_96_iso/keymap.c:32]: Yes, there should be.
image

Comment (noroadsleft) [reply layouts/default/96_iso/default_96_iso/keymap.c:32]: Are you saying you *want* them to be non-breaking?

Comment (fauxpark) [reply layouts/default/96_iso/default_96_iso/keymap.c:32]: That's what the rest of the keymap.c's do 🤷

**Comment by noroadsleft** [comment]
@fauxpark K, how is this now?

**Review by fauxpark — APPROVED** [review]

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**